src/ar.erl

```diff
@@ -141,7 +144,10 @@ main("") ->
 				)
 			},
 			{"enable (feature)", "Enable a specific (normally disabled) feature. For example, subfield_queries."},
-			{"disable (feature)", "Disable a specific (normally enabled) feature. For example, api_compat mode."}
+			{"disable (feature)", "Disable a specific (normally enabled) feature. For example, api_compat mode."},
+			{"use_tls", "Run the HTTP server under TLS"},
+			{"apex_domain (domain)", "Set the apex domain to use when running this node as a gateway"},
```

Comment:
Suggestion: "Enable gateway mode and use this as the apex domain name"

Reply:
So I'm a bit unsure about calling anything a _gateway mode_ as of yet. I also wanted to be more explicit and I was thinking of having another flag [`tx_origins`: Serve transaction contents on unique origins] to enable the behaviour in this PR, and having a bit of validation code fail early if TLS isn't enabled or an apex domain isn't specified. I also wonder if we should just enable TLS implicitly when that feature is enabled, but then again, could it make sense to run a node under HTTPS without this origins logic? Would other nodes be able to communicate with it?